specification/cognitiveservices/data-plane/ComputerVision/stable/v1.0/ComputerVision.json

```diff
@@ -895,7 +895,8 @@
       "type": "object",
       "properties": {
         "language": {
-          "$ref": "#/definitions/OcrResult"
+          "type": "string",
+          "description": "The BCP-47 language code of the text in the image."
```

Comment:
No the server for this type is unchanged for 2+ years.  The swagger was just wrong.

Reply:
A bit of background - up to now most of our customers have been using a hand-built .net client [SDK](https://github.com/Microsoft/Cognitive-Vision-Windows).  We're bringing the swagger to par so that this will be the official package source moving forward.

Reply:
@cthrash thank you Chris for clarifying.